src/dharma/DivBy_5_9.java

```diff
@@ -0,0 +1,22 @@
+package dharma;
+
+public class DivBy_5_9 {
+	
+	public void divNum(int start,int end) {
+		int number=0;
+		for(int i=start;i<=end;i++) {
```

Comment:
instead of travelling whole numbers for last number, you can reverse the loop means starts from last number, and you get directly last divisble number in limited iterations and then break the loop